tests/Http/Middleware/RateLimitTest.php

```diff
@@ -123,11 +123,12 @@ public function testRateLimitingFailsAndHeadersAreSetOnException()
                 return new Response('foo');
             });
         } catch (HttpException $exception) {
-            $this->assertSame(403, $exception->getStatusCode());
+            $this->assertSame(429, $exception->getStatusCode());
             $this->assertSame('You have exceeded your rate limit.', $exception->getMessage());
             $this->assertArrayHasKey('X-RateLimit-Limit', $exception->getHeaders());
             $this->assertArrayHasKey('X-RateLimit-Remaining', $exception->getHeaders());
             $this->assertArrayHasKey('X-RateLimit-Reset', $exception->getHeaders());
+            $this->assertArrayHasKey('Retry-After', $exception->getHeaders());
```

Comment:
@lucasmichot - Before i commit the assertion change, do you have an opinion on how i should perform the Retry-After assertion? Is the below snippet fine?

```php
try {
    $this->middleware->handle($request, function ($request) {
        return new Response('foo');
    });
} catch (HttpException $exception) {
    $this->assertSame(429, $exception->getStatusCode());
    $this->assertSame('You have exceeded your rate limit.', $exception->getMessage());
    $this->assertSame($exception->getHeaders()['X-RateLimit-Reset'] - time(), $exception->getHeaders()['Retry-After']);
    $this->assertArrayHasKey('X-RateLimit-Limit', $exception->getHeaders());
    $this->assertArrayHasKey('X-RateLimit-Remaining', $exception->getHeaders());
    $this->assertArrayHasKey('X-RateLimit-Reset', $exception->getHeaders());
}
```